packages/py-moose-lib/moose_lib/streaming/streaming_function_runner.py

```
@@ -55,6 +56,27 @@
+    _worker_exit_reason["reason"] = reason
+
+
+def _report_worker_exit() -> None:
+    try:
+        requests.post(
+            f"http://localhost:{moose_management_port}/metrics-logs",
+            json={"reason": _worker_exit_reason["reason"]},
+            timeout=0.5,
+        )
+    except Exception:
+        pass
```

Ruff (0.15.11)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)